cocotb/share/include/vpi_user_ext.h

```diff
@@ -0,0 +1,17 @@
+/* extensions to vpi_user.h */
```

Comment:
Please add the standard license header.

Reply:
Will do. I also have some other defines to add, but it'll be some days until I get to it.